lib/xcake/configuration/proxies/settings_proxy.rb

```diff
@@ -0,0 +1,9 @@
+module Xcake
+  class Configuration
+    class BuildSettingsProxy
+
+      def initialize(build_settings, key)
```

Comment:
Unused method argument - `build_settings`. If it's necessary, use `_` or `_build_settings` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.